compilers/concrete-optimizer/concrete-optimizer/src/optimization/dag/multi_parameters/partitionning.rs

```diff
@@ -333,6 +367,256 @@ pub mod tests {
         }
     }
 
+    fn get_external_partition_index_or(
+        external_partition: &ExternalPartition,
+        p_cut: &PartitionCut,
+        default: usize,
+    ) -> PartitionIndex {
+        for (i, e_partition) in p_cut.external_partitions.iter().enumerate() {
+            if external_partition == e_partition {
+                return PartitionIndex(i + p_cut.n_internal_partitions());
+            }
+        }
+        PartitionIndex(default)
+    }
+
+    pub const GLWE_PARAMS: GlweParameters = GlweParameters {
```

Comment:
maybe just prefix with TEST_  just to be sure is well used.

Reply:
Could be great also to name it like in tfhe-rs and give a pointer to that code. And maybe add more than one thfers partition to test?

Reply:
It's under the `tests` module already, so not sure it needs a prefix.
Partitioning isn't affected by those parameters, so we shouldn't need to try different ones. Optimization might need that in the future however.